src/components/shared/AdaptiveModal.module.css

```
@@ -0,0 +1,133 @@
+  box-shadow: var(--elevation-overlay);
+}
+
+.dialog[data-state="open"] {
+  animation: adaptiveDialogIn 150ms ease-out;
+}
+
+.dialog[data-state="closed"] {
+  animation: adaptiveDialogOut 120ms ease-in;
+}
+
+@keyframes adaptiveOverlayIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "adaptiveOverlayIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)